### PR TITLE
Add dataplane docs

### PR DESCRIPTION
**Pull request by fao89**
https://logserver.rdoproject.org/51/851/d054eec5ab28ee48bb5e4103546d1604a63673ed/github-check/openstack-operator-docs-preview/4fb110a/docs_build/docs_build/openstack/

### CODE REVIEW

File: .github/workflows/docs.yaml [hunk 1]
```diff
@@ -35,21 +35,23 @@ jobs:
...
 
       - name: Prepare gh-pages branch
         run: |
-          git restore docs/assemblies/custom_resources.adoc
           git config user.name github-actions
           git config user.email github-actions@github.com
 
           git branch -D gh-pages &>/dev/null || true
           git checkout -b gh-pages 4cd0193fc6c5bc7e76f3a0148d0447fb0d7fbe6a
+          git revert 4cd0193fc6c5bc7e76f3a0148d0447fb0d7fbe6a
```
Comment (jpodivin) [inline .github/workflows/docs.yaml:47]: Please add a comment about why are we reverting this.

**Review by jpodivin — CHANGES_REQUESTED** [review]

File: .github/workflows/docs.yaml (hunk 1 above)
Comment (fao89) [reply .github/workflows/docs.yaml]: it is just for creating an empty branch, but I found a better way

**Review by rebtoor — APPROVED** [review]

File: .github/workflows/docs.yaml (hunk 1 above)
Comment (rebtoor) [reply .github/workflows/docs.yaml]: ++ for `--orphan` usage

**Comment by fao89** [comment]
/test openstack-operator-build-deploy-kuttl
```
Waiting the cluster to be up
..........................................................................................make: *** [Makefile:2098: ceph] Error 1
```

**Comment by fao89** [comment]
recheck

**Pull request merged**